packages/frontend/src/components/ExportSelector/index.tsx

```diff
@@ -19,6 +25,13 @@ const ExportSelector: FC<
 
     const [exportType, setExportType] = useState<string | undefined>();
 
+    const { data } = useQuery<ApiDownloadCsv | undefined, ApiError>({
+        queryKey: [`google-sheets`],
+        enabled: false,
+    });
```

Comment:
This is the best way I could do to subscribe to `google-sheets` changes.

`useIsFetching` is not sufficient because we want to check:
* if query is `fetching`
* if query is `refetching`
* if query has `status` `started` and when that same satus changes to something that is not started. 

`enabled` is set to `false` so the query isn't run.